cli_helpers/tabular_output/output_formatter.py

```diff
@@ -179,7 +179,7 @@ def _get_type(self, value):
 
 
 def format_output(data, headers, format_name, **kwargs):
-    """Format output using *format_name*.
+    r"""Format output using *format_name*.
```

Comment:
I don't understand this change. Why is it better to convert the doc strings to be raw strings?

Reply:
Docstrings also emit warnings if they contain invalid sequences as noted in PR description during compilation/import and hence the tool made automatic changes to convert it into a raw string.

Reply:
Those docstring contains "invalid" backslash-sequence such as "\*".